baybe/searchspace/core.py

```diff
@@ -244,10 +243,10 @@ def contains_rdkit(self) -> bool:
         )
 
     @property
-    def param_bounds_comp(self) -> np.ndarray:
-        """Return bounds as tensor."""
-        return np.hstack(
-            [self.discrete.param_bounds_comp, self.continuous.param_bounds_comp]
+    def comp_rep_bounds(self) -> pd.DataFrame:
+        """The minimum and maximum values of the computational representation."""
+        return pd.concat(
```

Comment:
Would appreciate a comment here saying that this also follows the general guideline of having discrete parameters first and and continuous second.